src/docs/sanapi/available-metrics.md

```diff
@@ -65,6 +70,49 @@ Example:
 
 If you change the `aggregation` parameter to `AVG` it will return the average transaction volume over each 7 day interval. You can see how this API can be quite powerful and flexible.
 
+## histogramData
+
+A histogram is an approximate representation of the distribution of numerical or categorical data.
+
+This is better understood with an example: For the period of January 2018 - January 2019, fetch the amount of ether that was bought in every 100-dollar price range: $0-$100, $100-$200, ... , $1300-$1400.
```

Comment:
This is better understood with an example: For the period of January 2018 - January 2019, fetch the amount of Ether that was bought in every 100-dollar price range: $0-$100, $100-$200, ... , $1300-$1400.